src/workspace/mod.rs

```diff
@@ -488,3 +508,11 @@ impl Workspace {
         }
     }
 }
+
+fn draw_window(window:&WindowState, focused_window: Option<u32>){
+    if Some(window.window) == focused_window {
+        window.draw();
+    }else{
+        window.draw_without_border();
```

Comment:
all windows should have a border, focused or not.

The color of the border should differentiate.

I would postpone this though for now and implement this later.
The focusing could be given with a parameter. WDY think?

For now we could do black and white pixel for unfocused/focused?